src/Mount.cpp

```diff
@@ -2292,14 +2292,26 @@ void Mount::startSlewing(int direction)
 
             if (direction & EAST)
             {
-                LOG(DEBUG_STEPPERS, "[STEPPERS]: startSlewing(E): initial targetMoveTo is %l", -sign * 300000);
-                _stepperRA->moveTo(-sign * 300000);
+                // We need to subtract the distance tracked from the physical RA home coordinate
+                float trackedHours = (_stepperTRK->currentPosition() / _trackingSpeed) / 3600.0F;  // steps / steps/s / 3600 = hours
```

Comment:
Pull `trackedHours` out of the `if`, make it `const`

Reply:
Good catch